unixfs/ipld-merkledag/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2014-2018 Juan Batiz-Benet
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
```

Comment:
Do we need to copy this over, or can we just remove it?

We have https://github.com/ipfs/go-libipfs/blob/main/LICENSE.md

Reply:
I don't know, the license is different.
MIT != Apache 2 + MIT

I know they are compatible but I don't think we can just change the license like this (we can use apache2 + MIT license for changes we make to that code).

Maybe that fine idk.

Reply:
I've removed it, it's still in the git history.

Reply:
A few years ago I remember we made the licenses uniform across all repositories, but maybe some (like unixfs), got forgotten? Or maybe it was JS-land repos. Either way, I think this is something important to take into account during the migration. /cc @lidel do you have any input here?